extension/linkify-urls-in-code.js

```diff
@@ -27,11 +27,13 @@ window.linkifyURLsInCode = (() => {
 		$(codeBlobs)
 		.toArray()
 		.forEach(blob => {
-			const blobHTML = blob.innerHTML;
-			if (hasURL(blobHTML)) {
-				// Match URLs and remove < or > from beginning or end
-				const URLmatch = blobHTML.match(URLRegex)[0].replace(/(^&lt)|(&gt$)/, '');
-				blob.innerHTML = blobHTML.replace(URLmatch, linkifyURL(URLmatch));
+			if (hasURL(blob.innerHTML)) {
```

Comment:
FYI: Following @sindresorhus’s [earlier comment](https://github.com/sindresorhus/refined-github/pull/415#discussion_r115743702), we won’t have this in the future.